src/test/prfs_test.cpp

```diff
@@ -359,15 +359,15 @@ TEST(TestPRFs, TestPRFPKGadget)
 
     // h_i should equal:
     // 7ea1525fdbf9462c5144796937e1f80b9dad42369f7d4987c436b2f79257f9ac h_i =
-    // sha256( 0i00 || [a_sk]_252 || h_sig) a_sk:
+    // blake2s( 0i00 || [a_sk]_252 || h_sig) a_sk:
     // 0x0F000000000000FF00000000000000FF00000000000000FF00000000000000FF h_sig:
     // 0x0F000000000000FF00000000000000FF00000000000000FF00000000000000FF Note:
-    // This test vector has been generated by using the solidity sha256 function
-    // (we want to make sure that we generate the same digests both on-chain and
-    // off-chain))
+    // This test vector has been generated by using the solidity blake2s
```

Comment:
Have you generated this test vector by using a `blake2s` solidity implementation?

Reply:
No, I forgot to change the word "solidity" by hashlib (it should be the case in other files, I'll check and correct that).

Reply:
Thanks